kie-server-parent/kie-server-services/kie-server-services-kafka/src/main/java/org/kie/server/services/jbpm/kafka/KafkaServerExtension.java

```diff
@@ -201,21 +249,27 @@ public void onDeactivate(DeploymentEvent event) {
     }
 
     private void updateTopics(String deploymentId, ProcessDefinition processDefinition) {
-        updateTopics(topic2Signal, deploymentId, processDefinition.getSignalsDesc());
+        if (Boolean.getBoolean(SIGNALS_ENABLED)) {
```

Comment:
right now we have the metadata in the signal node regarding this so the behaviour should be like this.
if the signal map property is AUTO... we should MAP unless the implementation exists says otherthing than kafka
if it is none we should not map unless the property says kafka.

metadata overrides the flags.

Reply:
not including metadata at the moment